did-webkey/src/lib.rs

```diff
@@ -172,15 +150,9 @@ fn gpg_pk_to_vm(
 fn pk_to_vm_ed25519(
     did: &str,
     pk: sshkeys::Ed25519PublicKey,
-) -> Result<(VerificationMethodMap, DIDURL), String> {
-    let jwk = match ssh_pkk_to_jwk(&PublicKeyKind::Ed25519(pk)) {
-        Err(err) => return Err(format!("Unable to convert SSH key to JWK: {}", err)),
-        Ok(jwk) => jwk,
-    };
-    let thumbprint = match jwk.thumbprint() {
-        Err(err) => return Err(format!("Unable to calculate JWK thumbprint: {}", err)),
-        Ok(t) => t,
-    };
+) -> Result<(VerificationMethodMap, DIDURL)> {
+    let jwk = ssh_pkk_to_jwk(&PublicKeyKind::Ed25519(pk))?;
+    let thumbprint = jwk.thumbprint()?;
```

Comment:
Maybe it could be good to include the error strings using [context](https://docs.rs/anyhow/latest/anyhow/trait.Context.html#tymethod.context)?
https://docs.rs/anyhow/latest/anyhow/index.html#details
e.g.:
```rust
let thumbprint = jwk.thumbprint().context("Unable to calculate JWK thumbprint")?;
```

Reply:
Sure, we could. Did you want this anywhere else we calculate the JWK thumbprint also?

Reply:
Yes, that would make sense I think.